### ISSUE TITLE
Navigating the found keywords in Notebook Find is hard to follow

### ISSUE THREAD

**Issue by MsSQLGirl** [microsoft/azuredatastudio#14018]
**Is your feature request related to a problem? Please describe.**
When a "paragraph" finds a number of instances of the found keyword (using Notebook find), it's hard to know which one the arrow / navigation is up to. 

![image](https://user-images.githubusercontent.com/6405919/105437720-6d6fa000-5c16-11eb-91ff-8519958947f5.png)

**Describe the solution or feature you'd like**
Perhaps the current keyword that is activated by the arrow has a darker color or a different color to make it easier to distinguish.

**Describe alternatives you've considered**
none.

**Additional context**
Found this as a usability issue during Livesite discussion.

**Comment by radentonev** [comment]
@MsSQLGirl I think can follow what we have for MD files:

<img width="393" alt="Screen Shot 2021-01-22 at 10 50 14 AM" src="https://user-images.githubusercontent.com/71295692/105532744-19110280-5ca0-11eb-864d-08356d3d2876.png">


- Current match is highlighted in orange
- All other matches are highlighted in yellow

**Comment by chlafreniere** [comment]
This is a good one. Hopefully a quick fix on our side.

**Comment by MsSQLGirl** [comment]
Agreed. Thank you!

**Comment by MaddyDev** [comment]
@radentonev @yualan I have a fix that highlights the correct term one by one instead of highlight all results and then use different highlight color for the current one. Do you think there is added value to the latter way? (It's relatively easy for me to make that change as well) So asking for your opinions here. 
Also @chlafreniere  your thoughts as well?
